src/Microsoft.ApplicationInsights.AspNet/Extensibility/Implementation/Tracing/AspNetEventSource.cs

```diff
@@ -0,0 +1,121 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="AspNetEventSource.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+namespace Microsoft.ApplicationInsights.AspNet.Extensibility.Implementation.Tracing
+{
+    using System;
+    using System.Diagnostics.Tracing;
+
+    /// <summary>
+    /// Event source for Application Insights ASP.NET 5 SDK.
+    /// </summary>
+    [EventSource(Name = "Microsoft-ApplicationInsights-AspNet")]
```

Comment:
Should we call it AspNetCore? Without Core and without 5 that is quite generic...